packages/integration-sdk-cli/src/config.ts

```diff
@@ -61,7 +61,7 @@ async function isTypescriptPresent(
   // Refs:
   // - https://github.com/mrmlnc/fast-glob#pattern-syntax
   // - https://github.com/micromatch/micromatch#backslashes
-  const paths = await globby('**/*.ts', {
+  const paths = await globby(['**/*.ts', '!**/*.d.ts'], {
```

Comment:
Can you explain what motivated the change exactly?

Reply:
I sometimes want to run the debugger on the built version of graph projects. However, since the graph projects ship deceleration files, they always run with ts-node. It's interfered with the debugger. Additionally, using the CLI on built versions of projects shouldn't need to take the extra time and overhead of registering ts-node.